test/utils.ts

```diff
@@ -58,5 +60,11 @@ export function randomBigNumber() {
 }
 
 export function randomAddress() {
-  return ethers.utils.hexlify(ethers.utils.randomBytes(20));
+  return ethers.utils.getAddress(ethers.utils.hexlify(ethers.utils.randomBytes(20)));
+}
+
+export async function getParamType(contractName: string, functionName: string, paramName: string) {
```

Comment:
new function to return the types of a function's parameters. used when building Merkle trees.